{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2022-31667",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "partial"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-11-14T14:10:48.659302Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-11-14T14:10:52.881Z"
        }
      }
    ],
    "cna": {
      "title": "Harbor fails to validate the user permissions when updating a robot account",
      "source": {
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "format": "CVSS",
          "cvssV3_1": {
            "scope": "CHANGED",
            "version": "3.1",
            "baseScore": 6.4,
            "attackVector": "NETWORK",
            "baseSeverity": "MEDIUM",
            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:L/A:L",
            "integrityImpact": "LOW",
            "userInteraction": "NONE",
            "attackComplexity": "LOW",
            "availabilityImpact": "LOW",
            "privilegesRequired": "LOW",
            "confidentialityImpact": "NONE"
          },
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "affected": [
        {
          "vendor": "n/a",
          "product": "Harbor",
          "versions": [
            {
              "status": "affected",
              "version": "Harbor (Go) 2.x<=2.4.2; 2.5<=2.5.1"
            }
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "datePublic": "2022-08-30T21:00:00.000Z",
      "references": [
        {
          "url": "https://github.com/goharbor/harbor/security/advisories/GHSA-xx9w-464f-7h6f"
        }
      ],
      "x_generator": {
        "engine": "Vulnogram 0.2.0"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "Harbor fails to validate the user permissions when updating a robot account that\u00a0belongs to a project that the authenticated user doesn\u2019t have access to.\u00a0\n\nBy sending a request that attempts to update a robot account, and specifying a robot\u00a0account id and robot account name that belongs to a different project that the user\u00a0doesn\u2019t have access to, it was possible to revoke the robot account permissions.",
          "supportingMedia": [
            {
              "type": "text/html",
              "value": "<span style=\"background-color: rgb(255, 255, 255);\">\n\n<span style=\"background-color: rgb(255, 255, 255);\">\n\n<span style=\"background-color: rgb(255, 255, 255);\">\n\n<span style=\"background-color: rgb(255, 255, 255);\">Harbor fails to validate the user permissions when updating a robot account that&nbsp;</span><span style=\"background-color: rgb(255, 255, 255);\">belongs to a project that the authenticated user doesn\u2019t have access to.&nbsp;\n\n<span style=\"background-color: rgb(255, 255, 255);\">By sending a request that attempts to update a robot account, and specifying a robot&nbsp;</span><span style=\"background-color: rgb(255, 255, 255);\">account id and robot account name that belongs to a different project that the user&nbsp;</span><span style=\"background-color: rgb(255, 255, 255);\">doesn\u2019t have access to, it was possible to revoke the robot account permissions.</span>\n\n</span></span></span></span>",
              "base64": false
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-285",
              "description": "CWE-285"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "dcf2e128-44bd-42ed-91e8-88f912c1401d",
        "shortName": "vmware",
        "dateUpdated": "2024-11-14T11:50:48.289Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2022-31667",
    "state": "PUBLISHED",
    "dateUpdated": "2024-11-14T14:11:06.110Z",
    "dateReserved": "2022-05-25T23:31:47.418Z",
    "assignerOrgId": "dcf2e128-44bd-42ed-91e8-88f912c1401d",
    "datePublished": "2024-11-14T11:50:48.289Z",
    "assignerShortName": "vmware"
  },
  "dataVersion": "5.1"
}